style.css

```diff
@@ -47,47 +51,123 @@ header {
   position: relative;
 }
 
+
 .universe-background {
   position: absolute;
   width: auto;
-  margin-top: 0;
+  top: -10%;
   z-index: 0;
+  left: -55%;
 }
 
 .little-prince {
   width: 60%;
   z-index: 1;
+  position: absolute;
+  padding-top: 30%;
+  padding-left: 35%;
+}
+
+
+
+.sidenav {
+  height: 100%;
+  width: 0;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  background-color: #9591F2;
+  overflow-x: hidden;
+  transition: 0.5s;
```

Comment:
A good performance enhancement here would be to say what CSS properties to transition. For example, it looks like you're only really transitioning width here - so a good improvement would be to write something like `transition: width 0.5s ease;` or something similar. That means the browser will only try to animate that instead of all the CSS properties here, which saves a bit of processing!